Library/ViewModels/RewardCellViewModel.swift

```diff
@@ -57,21 +57,20 @@ RewardCellViewModelOutputs {
     }
     let projectAndReward = Signal.zip(project, reward)
 
-    self.conversionLabelHidden = project.map {
-      !needsConversion(projectCountry: $0.country, userCountry: AppEnvironment.current.config?.countryCode)
-    }
-
+    self.conversionLabelHidden = project.map(needsConversion(project:) >>> negate)
     self.conversionLabelText = projectAndRewardOrBacking
-      .filter { p, _ in
-        needsConversion(projectCountry: p.country, userCountry: AppEnvironment.current.config?.countryCode)
-      }
+      .filter(first >>> needsConversion(project:))
       .map { project, rewardOrBacking in
+        let (country, rate) = zip(
```

Comment:
Fun bit of FP: Optional zip coalescing to a default pair.

Reply:
Pretty cool!